src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.box {
+  display: flex;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.box:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.box:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.big-box {
+  height: 70px;
+}
+
+.big-icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input {
+  all: unset;
+  width: 100%;
+  font-family: "Avenir", Helvetica, sans-serif;
+  font-weight: 300;
+}
```

Comment:
you can make one selector for the searchbar and add the repeated rules there

```
.searchBar {
** all repeated styling + styling for big searchbar goes here **
}

.searchBar--small {
** here goes styling that differs **
}
```

Reply:
that's [checklist p.2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)